src/commands/testing/test.ts

```diff
@@ -345,6 +346,24 @@ export async function runTest(
     `Tests coverage report:
   ${coverageReportPath}`
   )
+
+  if (!ignoreFail) {
+    const failedTestsCount = testsCount - testsWithResultsCount
```

Comment:
`failedTestsCount` is equal to `testsWithoutResultCount `

Reply:
Not quite, because it could be for example:

testsCount: 10
testsWithResults: 5 (out of these 5, some could PASS and some could FAIL)
testsWithoutResults: 5 (would be 5 in this case)

Reply:
Now I see your point, indeed this is wrong.